lib/Sky/Api.php

```
@@ -356,6 +379,18 @@ public static function error($response_code, $error_code, $message, \Exception $
     }
 
     /**
+     * Gets the name of the method that corresponds to the given action name
+     * @param string $resource_name the name of the resource url
+     * @param string $action_name the alias of the method in the url
+     * @return string
+     */
+    protected function getMethodName($resource_name, $action_name) {
+        $action = $this->resources[$resource_name]['actions'][$action_name];
+        if (!$action) return false;
```

Should this return false or throw an exception if an action is not found?